.github/workflows/iceberg_spark_test.yml

```diff
@@ -156,3 +156,120 @@ jobs:
           ENABLE_COMET=true ENABLE_COMET_ONHEAP=true ./gradlew -DsparkVersions=${{ matrix.spark-version.short }} -DscalaVersion=${{ matrix.scala-version }} -DflinkVersions= -DkafkaVersions= \
             :iceberg-spark:iceberg-spark-runtime-${{ matrix.spark-version.short }}_${{ matrix.scala-version }}:integrationTest \
             -Pquick=true -x javadoc
+
+  iceberg-spark-rust:
+    if: contains(github.event.pull_request.title, '[iceberg]')
+    strategy:
+      matrix:
+        os: [ubuntu-24.04]
+        java-version: [11, 17]
+        iceberg-version: [{short: '1.8', full: '1.8.1'}, {short: '1.9', full: '1.9.1'}, {short: '1.10', full: '1.10.0'}]
+        spark-version: [{short: '3.5', full: '3.5.7'}]
```

Comment:
I feel we need a test for Spark 3.4
maybe we can skip for Iceberg 1.8 instead

Reply:
We don't currently test it, so it seems like a big can of worms to open. I can add a workflow and see how it goes.